connectors/analytics/conviva/src/main/java/com/theoplayer/android/connector/analytics/conviva/utils/Utils.kt

```diff
@@ -117,21 +147,19 @@ fun collectAdMetadata(ad: GoogleImaAd): ConvivaMetadata {
         // This tag must capture the "first" Ad Id in the wrapper chain when a Linear creative is available or there is
         // an error at the end of the wrapper chain. Set to "NA" if not available. If there is no wrapper VAST response
         // then the Ad Id and First Ad Id should be the same.
-        "c3.ad.firstAdId" to (ad.wrapperAdIds.firstOrNull() ?: ad.id),
+        "c3.ad.firstAdId" to ad.id,
 
         // [Preferred] Only valid for wrapper VAST responses.
         // This tag must capture the "first" Creative Id in the wrapper chain when a Linear creative is available or
         // there is an error at the end of the wrapper chain. Set to "NA" if not available. If there is no wrapper
         // VAST response then the Ad Creative Id and First Ad Creative Id should be the same.
-        "c3.ad.firstCreativeId" to validStringOrNA(
-            ad.wrapperCreativeIds.firstOrNull() ?: ad.creativeId
-        ),
+        "c3.ad.firstCreativeId" to "NA",
```

Comment:
We are never sending the `firstCreativeId` now?

Reply:
We will for google IMA ad, otherwise it is "NA" indeed since I don't think this info is present in the base `Ad` event. 
I split the previous `collectAdMetadata` function up into all the data we have on a base `Ad` object and the extra info we get from Google IMA. 
So after `collectAdMetadata` we check if we have an IMA ad, if so we call `updateAdMetadataForGoogleIma` and add the extra metadata.
If some of the information is present in the normal `Ad` object after all I can update it.